include/z64.h

```diff
@@ -223,14 +223,14 @@ typedef struct {
 
 typedef struct {
     /* 0x00 */ u8*      texture;
-    /* 0x04 */ s16      unk_4;
-    /* 0x06 */ s16      unk_6;
-    /* 0x08 */ u8       unk_8;
-    /* 0x09 */ u8       unk_9;
-    /* 0x0A */ u8       delayA;
-    /* 0x0B */ u8       delayB;
-    /* 0x0C */ s16      unk_C;
-    /* 0x0E */ s16      unk_E;
+    /* 0x04 */ s16      screenX;
+    /* 0x06 */ s16      screenY;
```

Comment:
should just be `y` imo, or you can do `xPos` and `yPos`

Reply:
didnt even realize that you had done that and had it closed. i did it independently, and def not trying to come off as stealing your effort (i am aware of the situation)

I'm down to change the names of x/y to whatever, is there somewhere else in the code that has already named screen coordinates to be consistent with?

Reply:
well either way I'm just glad it's being put in, you can look at the other stuff in there closed pr and use it if you want too. Usually everywhere else just simply uses x and y.